SPECS/gflags/gflags.spec

```diff
@@ -0,0 +1,64 @@
+Name:           gflags
+Summary:        The gflags package contains a C++ library that implements commandline flags processing. 
+Version:        2.2.2
+Release:        3%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://gflags.github.io/gflags/
+Source0:        https://github.com/gflags/gflags/archive/v%{version}.tar.gz
```

Comment:
Hi Pawel- is there a reason that Source0 has a non-standard archive name? I know this particular file downloads as `gflags-2.2.2.tar.gz`, and we usually comment out the download URL and set Source0 to %{name}-%{version}.tar.gz.

Reply:
Oh snap, thank you! That's a great catch. I've missed the fact that the files are downloaded under a different name.